python/servo/testing_commands.py

```diff
@@ -55,6 +55,8 @@
     "imports_granularity=Module",
     "--config",
     "group_imports=StdExternalCrate",
+    "--config",
+    "normalize_comments=true",
```

Comment:
It is very hard to review these changes because almost every file is touched. Consider moving all the formatting changes into their own commit. Is this the only non-formatting change?